pkg/cloud/api/mutable_resource.go

```diff
@@ -195,11 +195,41 @@ func (u *mutableResource[GA, Alpha, Beta]) CheckSchema() error {
 	if err != nil {
 		return err
 	}
+	ga, _ := u.ToGA()
+	alpha, _ := u.ToAlpha()
+	err = checkSubsetOf(ga, alpha)
+
+	if err != nil {
+		return fmt.Errorf("checkSubsetOf(ga, alpha) = %v, want nil", err)
```

Comment:
You should always print out the types %T so people have a straightforward time debugging this when it doesn't pass instead of just an error message with no context.

Reply:
done